driver/src/models/solution.rs

```diff
@@ -1,25 +1,34 @@
+use ethcontract::Address;
 use std::collections::HashMap;
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct ExecutedOrder {
+    pub account_id: Address,
```

Comment:
Would it make sure to contain the entire underlying Order in the struct? Not sure if there is a use case but would feel sensible and less repetitive from a modelling perspective. But then, it would complicate the solution parsing logic, as we have to find the orders in the list, so feel free to ignore.

Reply:
Overall I prefer the current approach. It only contains then necessary information even if some fields are the same as in another struct. It is easier when parsing the solution and a lot of test code would have to include information that is not needed.